README.md

```diff
@@ -1,679 +1,901 @@
 
-    Parameters:
+``` json
+"accuracy": {
+    ...
+}
+```
 
-        deployment_id: str - DataRobot deployment ID
+---
 
-    Sample config params:
+#### `GetBiasAndFairnessSettingsOperator`
 
-        "segment_analysis_enabled": True,
-        "segment_analysis_attributes": ['attribute1', 'attribute2'],
+Gets the Bias And Fairness settings for deployment.
 
-- `GetMonitoringSettingsOperator`
+Returns a dict with the [Bias And Fairness settings for a Deployment](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html#datarobot.models.Deployment.get_bias_and_fairness_settings)
 
-    Get monitoring settings for deployment.
+Parameters:
 
-    Parameters:
+| Parameter                   | Type | Description |
+|-----------------------------|------|-------------|
+| `deployment_id`             | str  | THe DataRobot deployment ID. |
 
-        deployment_id: str - DataRobot deployment ID
+No config params are required.
 
-    No config params are required.
+---
 
-    Returns a dict with the config params for a Deployment as follows:
+#### `UpdateBiasAndFairnessSettingsOperator`
 
-        {
-            "drift_tracking_settings": { ... } 
-            "association_id_settings": { ... }
-            "predictions_data_collection_settings": { ... }
-        }
+Updates the Bias And Fairness settings for deployment.
 
-    where: - drift_tracking_settings: [drift tracking settings for this deployment](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html#datarobot.models.Deployment.get_drift_tracking_settings)
-           - association_id_settings: [association ID setting for this deployment](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html#datarobot.models.Deployment.get_association_id_settings)
-           - predictions_data_collection_settings: [predictions data collection settings of this deployment](https://datarobot-public-api-client.readthedocs-hosted.com/en/latest-release/autodoc/api_reference.html?highlight=predictions_data_collection_settings#datarobot.models.Deployment.get_predictions_data_collection_settings)
-           
-- `UpdateMonitoringSettingsOperator`
+Parameters:
 
-    Updates monitoring settings for a deployment.
+| Parameter                   | Type | Description |
+|-----------------------------|------|-------------|
+| `deployment_id`             | str  | THe DataRobot deployment ID. |
 
-    Parameters:
+Sample config params:
 
-        deployment_id: str - DataRobot deployment ID
+``` json
+"protected_features": ['attribute1'],
+"preferable_target_value": 'True',
+"fairness_metrics_set": 'equalParity',
+"fairness_threshold": 0.1,
+```
```

Comment:
@oleksandr-saienko is this really `json` data?  because the syntax highlighter is marking this as bad syntax.

Reply:
Please check the other code snippets too as some of those seem to have bad syntax for JSON.

Reply:
@elatt It may not be. Feel free to remove or change the syntax highlighting if I'm wrong. This occurs where there's empty JSON as well.

Reply:
@elatt @elatt  I guess the problem with syntax highlighting related to using pythonic single quote ['] instead of general double quote ["], I will try to check this

Reply:
Is this text supposed to be json or is it supposed to be something else?